lib/pipeline/RequestHandler.js

```diff
@@ -26,7 +26,9 @@ class RequestHandler extends Duplex {
         type: request.type
       };
 
-      if (result && result.conformant != null) {
+      if (result &&
+        typeof result.conformant !== undefined &&
```

Comment:
if you use typeof the 'undefined' is an string, otherwise you could do direct comparison against undefined. i.e. either
`result.conformant !== undefined`

or 

`typeof result.conformant !== 'undefined'`